xlb/operator/boundary_condition/bc_regularized.py

```diff
@@ -253,6 +258,13 @@ def functional_pressure(
             # Find normal vector
             normals = get_normal_vectors(missing_mask)
 
+            # Find the value of rho from the missing directions
+            for q in range(wp.static(_q)):
```

Comment:
wp.static() only makes a difference if you have an expression instead of a simple constant.

Reply:
and here